2024-02-21 16:45:42,927 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-21 16:45:42,927 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Configure stats pid to 9200
2024-02-21 16:45:42,928 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-21 16:45:42,928 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-21 16:45:42,928 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-21 16:45:42,928 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-21 16:45:42,929 INFO    MainThread:9200 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240221_164542-xvsvfgax\logs\debug.log
2024-02-21 16:45:42,929 INFO    MainThread:9200 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240221_164542-xvsvfgax\logs\debug-internal.log
2024-02-21 16:45:42,929 INFO    MainThread:9200 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000297290F9060>
2024-02-21 16:45:42,930 INFO    MainThread:9200 [wandb_init.py:init():566] calling init triggers
2024-02-21 16:45:42,930 INFO    MainThread:9200 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'n_trees': '2', 'min_samples': '2', 'max_depth': '2', 'max_features': '0.5'}
2024-02-21 16:45:42,930 INFO    MainThread:9200 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 0indua2u
2024-02-21 16:45:42,931 INFO    MainThread:9200 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/0indua2u
2024-02-21 16:45:42,932 INFO    MainThread:9200 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-21 16:45:42,932 INFO    MainThread:9200 [wandb_run.py:_restore():2193] restore
2024-02-21 16:45:42,932 INFO    MainThread:9200 [wandb_run.py:_restore():2199] restore done
2024-02-21 16:45:47,589 INFO    MainThread:9200 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-21 16:45:47,590 INFO    MainThread:9200 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-21 16:45:47,594 INFO    MainThread:9200 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-21 16:45:47,704 INFO    MainThread:9200 [wandb_init.py:init():616] starting backend
2024-02-21 16:45:47,704 INFO    MainThread:9200 [wandb_init.py:init():620] setting up manager
2024-02-21 16:45:47,706 INFO    MainThread:9200 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-21 16:45:47,707 INFO    MainThread:9200 [wandb_init.py:init():628] backend started and connected
2024-02-21 16:45:47,715 INFO    MainThread:9200 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-21 16:45:47,715 INFO    MainThread:9200 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-21 16:45:47,715 INFO    MainThread:9200 [wandb_init.py:init():720] updated telemetry
2024-02-21 16:45:47,717 INFO    MainThread:9200 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-21 16:45:49,245 INFO    MainThread:9200 [wandb_run.py:_on_init():2262] communicating current version
2024-02-21 16:45:49,665 INFO    MainThread:9200 [wandb_run.py:_on_init():2271] got version response 
2024-02-21 16:45:49,666 INFO    MainThread:9200 [wandb_init.py:init():804] starting run threads in backend
2024-02-21 16:45:49,748 INFO    MainThread:9200 [wandb_run.py:_console_start():2241] atexit reg
2024-02-21 16:45:49,748 INFO    MainThread:9200 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-21 16:45:49,748 INFO    MainThread:9200 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-21 16:45:49,748 INFO    MainThread:9200 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-21 16:45:49,749 INFO    MainThread:9200 [wandb_init.py:init():847] run started, returning control to user process
2024-02-21 16:45:49,800 INFO    MainThread:9200 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-21 16:45:49,800 INFO    MainThread:9200 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-21 16:46:27,758 INFO    MainThread:9200 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-21 16:46:27,763 INFO    MainThread:9200 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-21 16:46:27,779 INFO    MainThread:9200 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-21 16:46:30,923 INFO    MainThread:9200 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-21 16:46:30,940 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-21 16:46:30,941 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Configure stats pid to 9200
2024-02-21 16:46:30,941 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-21 16:46:30,941 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-21 16:46:30,941 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-21 16:46:30,941 INFO    MainThread:9200 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-21 16:46:30,942 INFO    MainThread:9200 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240221_164630-495ulaps\logs\debug.log
2024-02-21 16:46:30,942 INFO    MainThread:9200 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240221_164630-495ulaps\logs\debug-internal.log
2024-02-21 16:46:30,942 INFO    MainThread:9200 [wandb_init.py:init():566] calling init triggers
2024-02-21 16:46:30,943 INFO    MainThread:9200 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'n_trees': '2', 'min_samples': '2', 'max_depth': '2', 'max_features': '0.5'}
2024-02-21 16:46:30,943 INFO    MainThread:9200 [wandb_init.py:init():591] re-initializing run, found existing run on stack: xvsvfgax
2024-02-21 16:46:30,944 INFO    MainThread:9200 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/xvsvfgax
2024-02-21 16:46:30,945 INFO    MainThread:9200 [jupyter.py:save_history():445] not saving jupyter history
2024-02-21 16:46:30,945 INFO    MainThread:9200 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-21 16:46:30,945 INFO    MainThread:9200 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-21 16:46:30,945 INFO    MainThread:9200 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-21 16:46:30,946 INFO    MainThread:9200 [wandb_run.py:_restore():2193] restore
2024-02-21 16:46:30,946 INFO    MainThread:9200 [wandb_run.py:_restore():2199] restore done
2024-02-21 16:46:35,416 INFO    MainThread:9200 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-21 16:46:35,417 INFO    MainThread:9200 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-21 16:46:35,421 INFO    MainThread:9200 [wandb_run.py:_footer_sync_info():3825] logging synced files
